pyproject.toml

```diff
@@ -124,3 +125,6 @@ packages = ["servicex"]
 
 [tool.coverage.run]
 dynamic_context = "test_function"
+
+[tool.pytest.ini_options]
+asyncio_mode = "auto"
```

Comment:
Not that I'm complaining, but did you find  you needed this?

Reply:
Pytest failed for `tests/test_servicex_adapter.py` with the following error:
[`E           ConnectionResetError: [Errno 104] Connection reset by peer`](https://github.com/ssl-hep/ServiceX_frontend/actions/runs/13773510333/job/38517405365#step:6:111)
Those two lines automatically add the _asyncio_ marker to all async test functions. I have to admit that I don't know whether the error was just an intermittent error and it can be fixed by this.